TACC/tup-ui
# fix: tup-334 app build requires core styles

❌ Not working yet. ❌

## Overview

Use `implicitDependencies` to force every app to build core-styles.

## Related

- [TUP-334](https://jira.tacc.utexas.edu/browse/TUP-334)

## Changes

- fix: add core-styles as an implicit dependency of each app

## Testing

1. Delete (if present) `/dist` and `/libs/core-styles/dist` and `/libs/core-styles/node_modules`.
2. Build any app, e.g.
    - `npx nx run tup-ui:build --skip-nx-cache`
    - `npx nx run tup-cms:build --skip-nx-cache`
    - `npx nx run ui-patterns:build --skip-nx-cache`
3. Verify the app builds without error finding a stylesheet.
